interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -710,6 +710,19 @@ public void gracefulShutdown() throws Exception {
     verifyNoMoreInteractions(responseObserver);
   }
 
+  /** Sends an rpc to an unimplemented method on the server. */
```

Comment:
So that all of the automatic tests (marked as @Test) stays together and in front of the manual tests.